README.md

```diff
@@ -56,6 +56,11 @@ npm install
 
 Once all dependencies are installed and your target development environments are setup (Xcode for iOS and Android Studio for Android), it should be possible to begin development with virtual devices.
 
+Before starting the application, make sure to build required HTML file for Camera QR Webview with the following command:
+```shell
+npm run build:webview
```

Comment:
Can we simply run this as a build task on iOS/Android?

Reply:
it's also required for development, so HTML file is needed before any actions.